eve/main.yml

```diff
@@ -2,8 +2,8 @@
 version: 0.2
 
 branches:
-  default:
-    stage: pre-merge
+  user/*, feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*, ft/*, bf/*, chore/*, docs/*:
```

Comment:
I think this is the common template used everywhere, but random question still, why including `user/` branches?

Reply:
It's just a default from the RelEng team